src/api/routes/tx.ts

```diff
@@ -250,6 +254,36 @@ export function createTxRouter(db: DataStore): express.Router {
     })
   );
 
+  router.get(
+    '/events',
+    asyncHandler(async (req, res, next) => {
+      const limit = parseTxQueryEventsLimit(req.query['limit'] ?? 96);
+      const offset = parsePagingQueryInput(req.query['offset'] ?? 0);
+
+      const { address, txId } = parseAddressOrTxId(req, res, next);
```

Comment:
Seems like `address` and `txId` input should be validated here for 400 (bad request) before handing off to datastore layer.